src/sidebar/components/Annotation/AnnotationDocumentInfo.js

```diff
@@ -1,12 +1,12 @@
 import propTypes from 'prop-types';
 
-import * as annotationMetadata from '../../helpers/annotation-metadata';
-
 /** @typedef {import("../../../types/api").Annotation} Annotation */
 
 /**
  * @typedef AnnotationDocumentInfoProps
- * @prop {Annotation} annotation - Annotation for which the document metadata will be rendered
+ * @prop {string} [domain]
+ * @prop {string} [link]
+ * @prop {string} title
```

Comment:
Can you add some brief descriptions of these props.

Reply:
Sure thing.